test/e2e/predicates.go

```diff
@@ -17,11 +17,14 @@ limitations under the License.
 package e2e
 
 import (
+	"fmt"
+	"github.com/kubernetes-sigs/kube-batch/pkg/apis/scheduling/v1alpha1"
```

Comment:
It is better to be put at a separate part.

Reply:
i didn't get this comment.
You mean we need a better alias name for this package?
i have added a new alias for this.

Reply:
sorry, it is better to be:

```
import (
	"fmt"
     
         . "github.com/onsi/ginkgo"
         . "github.com/onsi/gomega"

         "k8s.io/api/core/v1"
+        "k8s.io/apimachinery/pkg/api/resource"
          metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+        kubeletapi "k8s.io/kubernetes/pkg/kubelet/apis"
          schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"

+	 "github.com/kubernetes-sigs/kube-batch/pkg/apis/scheduling/v1alpha1"
)
```

Reply:
Done.
Thanks for the review.